resources/quality/ultimaker3/um3_aa0.25_PETG_Normal_Quality.inst.cfg

```diff
@@ -12,7 +12,7 @@ variant = AA 0.25
 weight = 0
 
 [values]
-material_print_temperature = =default_material_print_temperature - 5
+material_print_temperature = =default_material_print_temperature - 15
```

Comment:
```suggestion
material_print_temperature = =default_material_print_temperature - 10
```

Reply:
In the xml, I removed the `print temperature = 230` tags from the UM3/S3/5/7, and the global `print temperature` is set to 240. The AA0.25 0.1mm for UM3 and s-line should be at 225, so then the temperature offset should be -15.